concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2043,7 +2042,6 @@ jobs:
                     'CF_CLIENT_SECRET' => '${CF_CLIENT_SECRET}',
                     'CF_CLIENT_REDIRECT_URL' => 'https://billing.${SYSTEM_DNS_ZONE_NAME}/oauth/callback',
                     'CF_API_ADDRESS' => '${API_ENDPOINT}',
-                    'COMPOSE_API_KEY' => '${COMPOSE_API_KEY}',
```

Comment:
It seems that this is not sufficient to remove the env var from the app. Because this is using a simple `cf push` below and not a zero-downtime strategy, env vars won't get removed when they're removed from the manifest.

Reply:
Good spot. I've put a fix in which can be reverted once it reaches prod. However, in doing this I realised we are getting rid of Compose soon anyway, so I am not sure if we should even care.